src/python/graplctl/graplctl/docker_swarm_ops.py

```diff
@@ -0,0 +1,332 @@
+from __future__ import annotations
+
+import base64
+import json
+import logging
+import os
+import sys
+
+from typing import Dict, Iterator, List, NamedTuple, Optional, Set
+
+import boto3
+from botocore.exceptions import ClientError
+
+from mypy_boto3_ec2 import EC2ServiceResource
+from mypy_boto3_cloudwatch.client import CloudWatchClient
+from mypy_boto3_cloudwatch.type_defs import MetricTypeDef
+from mypy_boto3_sns.client import SNSClient
+from mypy_boto3_ssm import SSMClient
+
+from . import common
```

Comment:
I don't love `from .` but I'm willing to overlook it for now :)

Reply:
Likewise :p